docs/tutorials/essentials/part-8-rtk-query-advanced.md

```diff
@@ -0,0 +1,1092 @@
+
+:::
+
+## Managing Users Data
+
+We've finished converting our posts data management over to use RTK Query. Next up, we'll convert the list of users.
+
+Since we've already seen how to use the RTK Query hooks for fetching and reading data, for this section we're going to try a different approach. RTK Query's core API is UI-agnostic and can be used with any UI layer, not just React. Normally you should stick with using the hooks, but here we're going to work with the user data using _just_ the RTK Query core API so you can see how to use it.
+
+### Fetching Users Manually
+
+We're currently defining a `fetchUsers` async thunk in `usersSlice.js`, and dispatching that thunk manually in `index.js` so that the list of users is available as soon as possible. We can do that same process using RTK Query.
+
+We'll start by defining a `getUsers` query endpoint in `apiSlice.js`, similar to our existing endpoints. We'll export the `useGetUsersQuery` hook just for consistency, but for now we're not going to use it.
+
+```js title="features/api/apiSlice.js"
+export const apiSlice = createApi({
+  reducerPath: 'api',
+  baseQuery: fetchBaseQuery({ baseUrl: '/fakeApi' }),
+  tagTypes: ['Post'],
+  endpoints: builder => ({
+    // omit other endpoints
+
+    // highlight-start
+    getUsers: builder.query({
+      query: () => '/users'
+    })
+    // highlight-end
+  })
+})
+
+export const {
+  useGetPostsQuery,
+  useGetPostQuery,
+  // highlight-next-line
+  useGetUsersQuery,
+  useAddNewPostMutation,
+  useEditPostMutation
+} = apiSlice
+```
+
+If we inspect the API slice object, it includes an `endpoints` field, with one endpoint object inside for each endpoint we've defined.
+
+![API slice endpoint contents](/img/tutorials/essentials/api-slice-contents.png)
+
+Each endpoint object contains:
+
+- The same primary query/mutation hook that we exported from the root API slice object, but named as `useQuery` or `useMutation`
+- For query endpoints, an additional set of query hooks for scenarios like "lazy queries" or partial subscriptions
+- A set of ["matcher" utilities](https://redux-toolkit.js.org/api/matching-utilities) to check for the `pending/fulfilled/rejected` actions dispatched by requests for this endpoint
+- An `initiate` thunk that triggers a request for this endpoint
+- A `select` function that creates [memoized selectors](../../usage/deriving-data-selectors.md) that can retrieve the cached result data + status entries for this endpoint
+
+If we want to fetch the list of users outside of React, we can dispatch the `getUsers.initiate()` thunk in our index file:
+
+```jsx title="index.js"
+// omit other imports
+// highlight-next-line
+import { apiSlice } from './features/api/apiSlice'
+
+// Start our mock API server
+worker.start({ onUnhandledRequest: 'bypass' })
+
+// highlight-next-line
+store.dispatch(apiSlice.endpoints.getUsers.initiate())
```

Comment:
Maybe add an info box that describes that this would need to be unsubscribed or will never be cache-collected?